### ISSUE TITLE
unimported fct

### ISSUE THREAD

**Issue by joihn** [marcoaversa/diffinfinite#4]
thanks for this repo, 
This fct is non imported, thus undefined. 
https://github.com/marcoaversa/diffinfinite/blob/4d2309606e5ce6190828b96f436cfa962ec5d1ec/dm.py#L592

I guess one could import it from 
`dm_masks.py:75`

**Comment by marcoaversa** [comment]
Thanks for the feedback. We forgot to remove the normalise_to_neg_one_to_one function. In our case we manually normalise the latent by a factor instead of mapping the data into [-1,1]. This is because the data clipped in the VQVAE latent space were introducing biases in the colour of the synthetic images, we will fix it soon.

**Comment by joihn** [comment]
Thanks for the reply, 
I also noticed that my generated images are significantly brighter than the ground truth, maybe this could be the cause, 

Looking forward for the fix :) ! Do you have an ETA ?

**Comment by joihn** [comment]
actually it looks like the fix is actually already present:
![image](https://github.com/marcoaversa/diffinfinite/assets/11663917/b80912bc-ee5b-4506-9e18-1b3ea5015350)

![image](https://github.com/marcoaversa/diffinfinite/assets/11663917/cc460b5a-59a1-4bf6-9d5f-96eaba29c53e)

**Comment by marcoaversa** [comment]
Yep! The factor 50 should be adapted to your own dataset. In our case 50 was a good value to limit the noise to go inside [-1,1] on each step. I would expect that I can fix and polish it properly in 3 weeks.

**Comment by joihn** [comment]
okay thanks
